### PR TITLE
Badge and Button Component

### PR DESCRIPTION
**Pull request by iamharnad**
Added different types Badge and Button Components

1. Different types of badges
2. Different types of Buttons

### CODE REVIEW

File: Components/badge.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.badge{
+    padding: 0.2rem 0.65rem;
```
Comment (Prabhav-Sharma) [inline Components/badge.css:2]: Try to use globally defined variables for properties like margin, padding and gaps

**Review by ishapatell1 — APPROVED** [review]

File: Components/badge.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    color: white;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    font-size: .75em;
+    box-sizing: border-box;
+}
+
+.status-badge-online{
+    background-color: #1b7052;
```
Comment (parvez-rashid) [inline Components/badge.css:34]: Try to use variables here too, like you did above, for the sake of consistency

File: Components/badge.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+.status-badge-away{
+    background-color: #ff9204;
```
Comment (parvez-rashid) [inline Components/badge.css:37]: Try to use variables here too, like you did above, for the sake of consistency

File: Components/badge.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+}
+.status-badge-offline{
+    background-color: #dc3545;
```
Comment (parvez-rashid) [inline Components/badge.css:40]: Try to use variables here too, like you did above, for the sake of consistency

File: Components/badge.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+}
+.status-badge-number{
+    background-color: #ff3f6c;
```
Comment (parvez-rashid) [inline Components/badge.css:43]: Try to use variables here too, like you did above, for the sake of consistency

File: Components/button.css [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+  text-decoration: none;
+  display: inline-block;
+  padding: 0.5rem 1rem;
+  margin: 1.25rem 0;
+  border-radius: 0.3rem;
+  background-color: transparent;
+  border: none;
+  font-size: 1rem;
+  text-align: center;
+  outline: none;
```
Comment (parvez-rashid) [inline Components/button.css]: Avoid using `outline: none` as it is bad for accessibility.

**Review by parvez-rashid — APPROVED** [review]
Have added my suggestions as comments. All the best!

File: Components/badge.css (hunk 1 above)
Comment (iamharnad) [reply Components/badge.css:2]: Sure, i will make changes

File: Components/badge.css (hunk 2 above)
Comment (iamharnad) [reply Components/badge.css:34]: sure, thank you

**Pull request merged**